src/Features/VisualBasic/Portable/InitializeParameter/InitializeParameterHelpers.vb

```diff
@@ -13,11 +13,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InitializeParameter
                    TypeOf node Is LambdaExpressionSyntax
         End Function
 
-        Public Shared Function GetBlockOperation(functionDeclaration As SyntaxNode, semanticModel As SemanticModel, cancellationToken As CancellationToken) As IBlockOperation
-            Dim operation = semanticModel.GetOperation(functionDeclaration, cancellationToken)
-            Return If(TypeOf operation Is IAnonymousFunctionOperation,
-                DirectCast(operation, IAnonymousFunctionOperation).Body,
-                DirectCast(operation, IBlockOperation))
+        Public Shared Function CanOfferRefactoring(functionDeclaration As SyntaxNode, operation As IOperation) As Boolean
+            Return operation.Kind = OperationKind.AnonymousFunction Or operation.Kind = OperationKind.Block
```

Comment:
Syntax node is unused, so seems the API is language agnostic. Can you instead move this code directly into `AbstractInitializeParameterCodeRefactoringProvider.cs`?